src/v2/Apps/getAppNovoRoutes.tsx

```diff
@@ -36,6 +38,9 @@ export function getAppNovoRoutes(): RouteConfig[] {
       {
         routes: artworkRoutes,
       },
+      {
+        routes: auctionsRoutes,
+      },
```

Comment:
Should this route be reflected in `webpackPublicPath` too (or only later when `/auctions2` replaces `/auctions`)?

Reply:
It doesn't look like this was `converted: true`, seems like an unrelated change to keep the routes in sync.

Reply:
Yep 👍